packages/components/src/slot-fill/context.js

```diff
@@ -120,14 +119,48 @@ class SlotFillProvider extends Component {
 		}
 	}
 
+	triggerListeners() {
+		this.listeners.forEach( ( listener ) => listener() );
```

Comment:
Isn't this updating much more than we were previously, where previously we were limiting to triggering updates only within the fills for a given slot?

Reply:
yes but should we be more granular. I mean the `useSlot` hook only rerenders the fills where the slot changes, so I don't really think it makes a lot of difference.